modules/javafx.graphics/src/main/native-glass/mac/GlassWindow.m

```diff
@@ -843,7 +843,10 @@ static jlong _createWindowCommonDo(JNIEnv *env, jobject jWindow, jlong jOwnerPtr
         }
         else
         {
-            [window->nsWindow performSelectorOnMainThread:@selector(setContentView:) withObject:nil waitUntilDone:YES];
+            // If the contentView is set to nil within performKeyEquivalent: the OS will crash.
+            NSView* dummy = [[NSView alloc] initWithFrame: NSMakeRect(0, 0, 10, 10)];
+            [window->nsWindow performSelectorOnMainThread:@selector(setContentView:) withObject:dummy waitUntilDone:YES];
+            [dummy release];
```

Comment:
This looks fine, given that we wait until the previous call to `performSelectorOnMainThread` is done.

Reply:
For what it's worth I did complete a full run of robot-based system tests.